index.html

```diff
@@ -1080,8 +1080,17 @@ <h4>Proof Configuration (Ed25519Signature2020)</h4>
   <var>options</var>.<var>proofPurpose</var>.
               </li>
               <li>
-  Return <var>proofConfig</var>.
+  Set <var>proofConfig</var>.<var>@context</var> to
+  <var>unsecuredDocument</var>.<var>@context</var>
               </li>
+              <li>
+  Let <var>canonicalProofConfig</var> be the result of applying the
+  Universal RDF Dataset Canonicalization Algorithm
+  [[RDF-CANON]] to the <var>proofConfig</var>.
```

Comment:
```suggestion
  Let <var>canonicalProofConfig</var> be the result of applying the
  Universal RDF Dataset Canonicalization Algorithm
  [[RDF-CANON]] to the <var>proofConfig</var>.
```

Good catch! Note that a similar change will need to be made to the jcs-eddsa-2022 and ecdsa-2019 algorithms as well.